src/components/views/elements/DesktopBuildsNotice.tsx

```diff
@@ -37,19 +37,22 @@ export default function DesktopBuildsNotice({ isRoomEncrypted, kind }: IProps) {
     if (EventIndexPeg.get()) return null;
 
     if (EventIndexPeg.error) {
-        return <>
-            { _t("Message search initialisation failed, check <a>your settings</a> for more information", {}, {
-                a: sub => (<a onClick={(evt) => {
-                    evt.preventDefault();
-                    dis.dispatch({
-                        action: Action.ViewUserSettings,
-                        initialTabId: UserTab.Security,
-                    });
-                }}>
-                    { sub }
-                </a>),
-            }) }
-        </>;
+        return (
+            <div className="mx_DesktopBuildsNotice">
+                { _t("Message search initialisation failed, check <a>your settings</a> for more information", {}, {
+                    a: sub => (<a href=""
```

Comment:
I think `href=""` doesn't need to be there at all

Reply:
Without it the text does not change color
![image](https://user-images.githubusercontent.com/9784257/135762270-7a5e9a90-a444-4df2-aa88-f783d1b7f16f.png)

Reply:
Ah, sorry, we have a class for this, should be `className="mx_linkButton"`

Reply:
Ah ok switched to using that now.